composer.json

```diff
@@ -14,10 +14,16 @@
       "src/"
     ]
   },
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/OnTheGoSystems/collect.git"
+    }
+  ],
```

Comment:
@BruceOnTheGo if in the composer.jon of https://github.com/OnTheGoSystems/collect.git you change the name from `tightenco/collect` to `otgs/collect`, I can quickly add it to Packagist, then you can remove this entry (it speeds up Composer).

Reply:
@andreasciamanna Can you do the change in the collect repository? Then I'll adjust the other plugins